### PR TITLE
global: migration to invenio-records-rest

### PR DESCRIPTION
**Pull request by egabancho**
Supersedes #6

### CODE REVIEW

**Comment by egabancho** [comment]
It needs https://github.com/inveniosoftware/invenio-records-rest/pull/24

File: examples/app.py [hunk 1]
```diff
@@ -34,17 +34,24 @@
...
    $ npm install -g node-sass clean-css requirejs uglify-js
    $ cd examples
+   $ pip install -r requirements.txt
    $ flask -a app.py npm
    $ cd static ; npm install ; cd ..
    $ flask -a app.py collect -v
    $ flask -a app.py assets build
 
+Make sure that ``elasticsearch`` server is running:
+.. code-block:: console
```
Comment (jirikuncar) [inline examples/app.py]: can you put empty lines around for better readability?

Comment (egabancho) [inline examples/app.py]: :+1:

File: examples/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
```
Comment (jirikuncar) [inline examples/requirements.txt]: 2016

File: invenio_search_ui/static/scss/invenio_search_ui/search.scss [hunk 3]
```diff
@@ -28,12 +28,15 @@ $invenio-search-xlarge-space: 20px !default;
 $invenio-search-grey: #eee !default;
 $invenio-search-light-grey: #fafafa !default;
 
+.invenio-search-count {
```
Comment (jirikuncar) [inline invenio_search_ui/static/scss/invenio_search_ui/search.scss]: update copyright `, 2016`

File: examples/requirements.txt (hunk 2 above)
Comment (egabancho) [inline examples/requirements.txt]: This reminds me that I need to update my snippets :relaxed:

**Pull request merged**